example/tools/BUILD

```diff
@@ -131,6 +131,7 @@ shellcheck_binary(name = "shellcheck")
 
 multi_formatter_binary(
     name = "format",
+    clangformat = "@llvm_toolchain_llvm//:bin/clang-format",
```

Comment:
ah, that's so convenient that the toolchain already includes the format binary 😌

Reply:
It also includes `clang-tidy` so paves the way for including linting too.